File: transektcount/src/main/java/com/wmstein/transektcount/database/Meta.java
package com.wmstein.transektcount.database;

/*
 * Created by wmstein on 31.03.2016.
 */
public class Meta
{
    public int id;
    public int temp;
    public int wind;
    public int clouds;
    public String start_tm;
    public String end_tm;

    //setting M_ID *******************
    public void setM_ID (int M_id)
    {
        this.id = M_id;
    }
    
    //setting M_TEMP
    public void setM_TEMP(int M_temp)
    {
        this.temp = M_temp;
    }
    
    //setting M_WIND
    public void setM_WIND(int M_wind)
    {
        this.wind = M_wind;
    }

    //setting M_CLOUDS
    public void setM_CLOUDS(int M_clouds)
    {
        this.clouds = M_clouds;
    }

    //setting M_START_TM
    public void setM_START_TM(String M_start_tm)
    {
        this.start_tm = M_start_tm;
    }

    //setting M_END_TM
    public void setM_END_TM(String M_end_tm)
    {
        this.end_tm = M_end_tm;
    }

}

File: transektcount/src/main/java/com/wmstein/transektcount/database/Meta.kt
package com.wmstein.transektcount.database

/********************************************************
 * Table Meta interface with record of transect metadata
 * Created by wmstein on 31.03.2016.
 * Converted to Kotlin on 2024-03-09,
 * last edited on 2026-02-21.
 */
class Meta {
    @JvmField
    var id = 0
    @JvmField
    var temps = 0
    @JvmField
    var tempe = 0
    @JvmField
    var winds = 0
    @JvmField
    var winde = 0
    @JvmField
    var clouds = 0
    @JvmField
    var cloude = 0
    @JvmField
    var date: String? = null     // Formet: EN: yyyy-mm-dd, DE: dd.mm.yyyy
    @JvmField
    var start_tm: String? = null // Format: hh:mm
    @JvmField
    var end_tm: String? = null   // Format: hh:mm
    @JvmField
    var note: String? = null
}